cmd/queue/main.go

```diff
@@ -292,11 +292,16 @@ func buildServer(ctx context.Context, env config, healthState *health.State, rp
 	breaker := buildBreaker(logger, env)
 	metricsSupported := supportsMetrics(ctx, logger, env)
 	tracingEnabled := env.TracingConfigBackend != tracingconfig.None
+	concurrencyStateEnabled := env.ConcurrencyStateEndpoint != ""
 	timeout := time.Duration(env.RevisionTimeoutSeconds) * time.Second
 
 	// Create queue handler chain.
 	// Note: innermost handlers are specified first, ie. the last handler in the chain will be executed first.
 	var composedHandler http.Handler = httpProxy
+	if concurrencyStateEnabled {
```

Comment:
so the concurrencyStateEnabled don't need to be "freeze-proxy"?

Reply:
Correct, it should be the freeze service endpoint/port.